pkg/mountkexec/mountkexec.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mountkexec
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/u-root/u-root/pkg/loop"
+	"github.com/u-root/u-root/pkg/mount"
+	"golang.org/x/sys/unix"
+)
+
+// MountISO runs a command with mount,unix and loop
```

Comment:
This comment needs more information. What does it do? 
Looks like mount the iso path via the loopback device? It does not appear to run a command, though.
Can you make this a bit more informative?

Reply:
Done.